### PR TITLE
Ben/voting

### PR DESCRIPTION
**Pull request by beshaya**

### CODE REVIEW

File: src/data/prepopulated.ts [hunk 1]
```diff
@@ -182,6 +182,19 @@ export const prepopulated: {
...
   },
 
+  'Voting in-person': {
+    riskProfile: 'average',
+    interaction: 'oneTime',
+    personCount: 2,
+
+    setting: 'indoor',
+    distance: 'sixFt',
+    duration: 10,
```
Comment (blanchardjeremy) [inline src/data/prepopulated.ts:192]: Hmmm. I guess it depends on what kind of voting we're talking about. A ballot drop-off or filling out a whole ballot. What had you think 10 minutes?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@ name: CI
 # events but only for the master branch
 on:
   push:
-    branches: [ main ]
+    branches: [ main, develop ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, develop ]
```
Comment (blanchardjeremy) [inline .github/workflows/ci.yml:9]: Your #264 PR seems to be mixed in with this one. Do you wanna separate them out?

**Review by blanchardjeremy — CHANGES_REQUESTED** [review]
Looks good overall. Take a look at the two comments and let me know what you think.

File: src/data/prepopulated.ts (hunk 1 above)
Comment (oremanj) [reply src/data/prepopulated.ts:192]: I think 10min is a reasonable estimate for how long it takes to sign the registry, get your ballot, and fill it out. Plus whatever part of the line is indoors, if there's a line.

Comment (beshaya) [reply src/data/prepopulated.ts:192]: I just copied what Josh said in the bug. Do you think the numbers should be different? I haven't voted in person in a loooong time.

I'd be happy to add a mail-in dropoff scenario as well.

Comment (blanchardjeremy) [reply src/data/prepopulated.ts:192]: I don't know if I've ever voted in-person, so I'm not a reliable source, either. Let's keep it at 10, then!

Seems like a ballot drop-off wouldn't be all that useful, given that this is already so few points.

Comment (oremanj) [reply src/data/prepopulated.ts:192]: I think mail-in dropoff is basically free.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (blanchardjeremy) [reply .github/workflows/ci.yml]: I went ahead and removed the devleop branch stuff myself.

**Comment by blanchardjeremy** [comment]
@oremanj I say this is good to go!

**Review by blanchardjeremy — APPROVED** [review]
Good to go!

**Pull request merged**